pyabc/external/r_rpy2.py

```diff
@@ -11,20 +11,21 @@
     See also :ref:`installation of optional dependencies <install-optional>`.
 
 .. note::
-
-    Support of R via rpy2 is considered experimental. Currently, an rpy2
-    version <= 2.9.5 is required. Versions >= 3.0.0 do not work yet properly
-    for various reasons (see #116). R versions 3.4 and 3.5 have shown to work.
+    Support of R via rpy2 is considered experimental for various reasons
+    (see #116).
     Should this not work on your system, consider accessing R script-based.
 """
 
 try:
+    import rpy2.robjects
```

Comment:
why two imports for the same library?

Reply:
The second one is just as a convenience in order ton not always have to write ``rpy2.robjects.r`` but just ``r``. But I will simplify that now, actually right.